service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/ShareCaseController.java

```diff
@@ -45,9 +56,52 @@ public void shareCase(@PathVariable("caseId") String caseId, @RequestBody Users
                 new CaseUser(userId, CASE_ROLES)));
     }
 
+    @PostMapping(value = "/support/cases/share", consumes = APPLICATION_JSON_VALUE)
+    @Secured("caseworker-publiclaw-systemupdate")
+    public void shareCases(@Valid @RequestBody ShareLocalAuthorityCases shareCases) {
+
+        final String query = new JSONObject()
+            .put("from", 0)
+            .put("size", 1000)
+            .put("query", of("match", of("data.caseLocalAuthority", shareCases.localAuthority)))
```

Comment:
Would it be better to use a bool query here? https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-bool-query.html

Reply:
something like
```json
{
  "query": {
    "bool" : {
      "must" : {
        "term" : { "data.caseLocalAuthority" : "LA" }
      }
    }
  }
}
```

Reply:
fair enough, was just copying their example to our fields